sphinx_gallery/gen_gallery.py

```diff
@@ -191,32 +184,34 @@ def sumarize_failing_examples(app, exception):
         return
 
     gallery_conf = app.config.sphinx_gallery_conf
-    failing_examples = set([os.path.normpath(path) for path in
-                            gallery_conf['failing_examples']])
-    expected_failing_examples = set([os.path.normpath(path) for path in
+    failing_examples = {os.path.normpath(path): traceback
```

Comment:
Yes this is entirely related to the PR. The thing is now I pass the absolute path of the files during execution. But at the end when I verify which examples failed I'm matching strings containing the path and those paths must be equal and thus can't have the `../` entries that tend to appear when joining relative paths(Which is how we configure the gallery, relative to `conf.py`) so the way I found to fix this is to normalize the paths at the end.